snorkel/contrib/fonduer/fonduer/parser.py

```diff
@@ -135,7 +135,7 @@ def __init__(self,
         for (pattern, replace) in replacements:
             self.replacements.append((re.compile(pattern, flags=re.UNICODE), replace))
         if self.lingual:
-            self.batch_size = 7000  # TODO(bhancock8): what if this is smaller than a block?
+            self.batch_size = 7000
```

Comment:
Rather than just get rid of this comment, it would probably be good to leave a comment for why `7000` is a magic number here.

Reply:
Yes, that's the limit for CoreNLP